### DISCUSSION TITLE
Deduplicate compiler warnings in CI / code review

### DISCUSSION THREAD

**Discussion by piegamesde** [community/community#16661] [Pull Requests]
![image](https://user-images.githubusercontent.com/14054505/168493847-bb1e2f67-c062-4e51-9c81-f081cc634cbc.png)

Identical compiler warnings and other CI notes should be deduplicated across runs. Additionally, there should be an option to turn them off.

It is common to build the project multiple times in CI, for example for different platforms, with different toolchains/version or different configurations. With this, just a handful of compile warnings can easily drown the view with unhelpful junk, making reviewing a chore.

**Comment by ivan-gaydamakin** [comment]
any updates?

**Comment by ZehMatt** [comment]
🙃
